server/test/integration/com/thoughtworks/go/config/GoConfigMigrationIntegrationTest.java

```diff
@@ -1450,6 +1455,187 @@ public void shouldConvertPluginToPluginIdOnConfigRepoAsPartOfMigration95() throw
+
+        String migratedContent = migrateXmlString(configXml, 97);
+
+        assertThat(migratedContent, not(containsString("<filter>")));
+        assertThat(migratedContent, not(containsString("dest='dest'")));
+        assertThat(migratedContent, not(containsString("autoUpdate='false'")));
+        assertThat(migratedContent, not(containsString("invertFilter='true'")));
+    }
+
+    @Test
+    public void shouldRemoveUncesseryMaterialFieldsFromSvnMaterialAsPartOfMigration98() throws Exception {
+        String configXml = "<cruise schemaVersion='96'>" +
+                "<config-repos>\n" +
+                "   <config-repo pluginId=\"json.config.plugin\" id=\"config-repo-1\">\n" +
+                "      <svn url=\"test-repo\" dest='dest' autoUpdate='false' checkexternals='false' invertFilter='true' materialName=\"foo\">\n" +
+                "        <filter>\n" +
+                "          <ignore pattern=\"asdsd\" />\n" +
+                "        </filter>\n" +
+                "      </svn>" +
+                "   </config-repo>\n" +
+                "</config-repos>" +
+                "</cruise>";
+
+        assertThat(configXml, containsString("<filter>"));
+        assertThat(configXml, containsString("dest='dest'"));
+        assertThat(configXml, containsString("autoUpdate='false'"));
+        assertThat(configXml, containsString("invertFilter='true'"));
+
+        String migratedContent = migrateXmlString(configXml, 97);
+
+        assertThat(migratedContent, not(containsString("<filter>")));
+        assertThat(migratedContent, not(containsString("dest='dest'")));
+        assertThat(migratedContent, not(containsString("autoUpdate='false'")));
+        assertThat(migratedContent, not(containsString("invertFilter='true'")));
+    }
+
+    @Test
+    public void shouldRemoveUncesseryMaterialFieldsFromp4MaterialAsPartOfMigration98() throws Exception {
+        String configXml = "<cruise schemaVersion='96'>" +
+                "<config-repos>\n" +
+                "   <config-repo pluginId=\"json.config.plugin\" id=\"config-repo-1\">\n" +
+                "      <p4 port=\"10.18.3.241:9999\" username=\"cruise\" password=\"password\" autoUpdate='false' invertFilter='true' dest=\"dest\">\n" +
+                "          <view><![CDATA[//depot/dev/... //lumberjack/...]]></view>\n" +
+                "        <filter>\n" +
+                "          <ignore pattern=\"asdsd\" />\n" +
+                "        </filter>\n" +
+                "      </p4>" +
+                "   </config-repo>\n" +
+                "</config-repos>" +
+                "</cruise>";
+
+        assertThat(configXml, containsString("dest=\"dest\""));
+        assertThat(configXml, containsString("<filter>"));
+        assertThat(configXml, containsString("autoUpdate='false'"));
+        assertThat(configXml, containsString("invertFilter='true'"));
+
+        String migratedContent = migrateXmlString(configXml, 97);
```

Comment:
I guess you need to update all the tests to use schemaVersion 99 or whatever is available when you next rebase, and trigger the PR pipeline. This test must be failing currently since the version that migrates these nodes in your PR is 98 not 97 :)